client/src/pages/Initial/LandingPageSherry/LandingPageSherry.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useState, useEffect, useContext } from 'react';
+import PropTypes from 'prop-types';
+import './LandingPageSherry.scss';
+import InstagramLogo from '../../../assets/social/instagram_icon.png';
```

Comment:
uncommit this file as well since it's not part of your current issue, typically this happens when you make a branch while you're still on a branch... so whenever you start a new issue make sure to do the following....

```
git switch dev
git pull
git checkout -b <branch>
```

switching back to dev our main branch, and pulling will make sure that you have the most updated version on your local machine ;)